app/src/main/res/layout/fragment_main.xml

```diff
@@ -0,0 +1,12 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activities.MainActivity">
+
+    <ListView
+        android:id="@+id/movies_list_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
```

Comment:
Given the requeriments of the code challenge, Do you think ListView is the best option to solve the problem following the android architecture guide?

Reply:
I think RecyclerListView is a better option.